cmd/genkubedocs/postprocessing.go

```diff
@@ -27,7 +27,7 @@ import (
 // MarkdownPostProcessing goes though the generated files
 func MarkdownPostProcessing(cmd *cobra.Command, dir string, processor func(string) string) error {
 	for _, c := range cmd.Commands() {
-		if !c.IsAvailableCommand() || c.IsHelpCommand() {
+		if !c.IsAvailableCommand() || c.IsAdditionalHelpTopicCommand() {
```

Comment:
cobra changed ?  I think we could have three commit, 1.pflag gen, 2.cobra gen, 3.the code you changed

Reply:
Yes, method rename. Please see my [last commit](https://github.com/kubernetes/kubernetes/pull/53631/commits/f31ec301c36ffac215242fb9bf44b9e0633d9a8c).